spec/unit/lib/cloud_controller/config_spec.rb

```diff
@@ -585,6 +585,63 @@ module VCAP::CloudController
       end
     end
 
+    describe 'broker_client_async_poll_exponential_backoff_rate' do
+      let(:backoff_rate) { 1.0 }
+      let(:schema_context) { :worker }
```

Comment:
These two `let`s are overwritten by the individual tests. Would it make sense to omit them here or set them to `nil`?